com/win32comext/axdebug/util.py

```diff
@@ -34,23 +34,12 @@ def trace(*args):
 # (Now this is only true for Document objects, and Python
 # now does ensure this.
 
+def _wrap(object, iid):
+    useDispatcher = None
+    if debugging:
+        from win32com.server.policy import DispatcherWin32trace as useDispatcher
```

Comment:
Does it matter much to delay this import to if and when wrap is called in debug? Otherwise I'd move the import to the top to further simplify like in `scriptdispatch`